src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends FigureColor {
+    private int radius;
+
+    public Circle(int radius, String color) {
```

Comment:
Passing raw values to the constructor can be less readable than using a variable. Consider assigning 'radius' and 'color' to variables before passing them to the constructor.